scripts/prepdocs.py

```diff
@@ -245,7 +254,9 @@ async def main(strategy: Strategy, credential: AsyncTokenCredential, args: Any):
     )
     parser.add_argument("files", nargs="?", help="Files to be processed")
     parser.add_argument(
-        "--datalakestorageaccount", required=False, help="Optional. Azure Data Lake Storage Gen2 Account name"
+        "--datalakestorageaccount",
```

Comment:
Do you know what led to the change of formatting of these lines? Is your black configured differently?

Reply:
I'm not bothered by the formatting itself, but it's nice to avoid formatting changes in unrelated PRs to reduce git noise.

Reply:
See this:
https://github.com/Azure-Samples/azure-search-openai-demo/pull/1351#issuecomment-1972104927

Reply:
I'm working inside github codespaces.
It's something related to line length but even setting the line length the closest I got to no change is by setting the line length to 90 only one line was changed